client/src/components/Layout/Navbar.tsx

```diff
@@ -0,0 +1,217 @@
+import { useState } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import MenuList from '@material-ui/core/MenuList';
+import MenuItem from '@material-ui/core/MenuItem';
+import Popover from '@material-ui/core/Popover';
+import { Link } from 'react-router-dom';
+import Toolbar from '@material-ui/core/Toolbar';
+import profileImage from '../../Images/775db5e79c5294846949f1f55059b53317f51e30.png';
+import logo from '../../Images/logo.png';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { useAuth } from '../../context/useAuthContext';
+
+// Placeholders simulating data from DB
+const notifications: ActiveNotification[] = [
+  {
+    title: 'Notification 1 Title',
+    subtitle: 'Notification 1 Subtitle',
+    date: '9/9/2000',
+  },
+  {
+    title: 'Notification 2 Title',
+    subtitle: 'Notification 2 Subtitle',
+    date: '8/8/2000',
+  },
+];
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
You will notice in this codebase, we follow a pattern where the styling is kept in a separate file